password_security/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 
     'name': 'Password Security',
     "summary": "Allow admin to set password security requirements.",
-    'version': '9.0.1.0.3',
+    'version': '9.0.1.0.4',
```

Comment:
As this requires module update, it should be 9.0.1.1.0. You can squash already the commits as the history is very simple.